tests/cache/tests.py

```diff
@@ -1398,6 +1398,20 @@ def test_pylibmc_options(self):
         self.assertTrue(cache._cache.binary)
         self.assertEqual(cache._cache.behaviors['tcp_nodelay'], int(True))
 
+    def test_default(self):
```

Comment:
I'd rather put this in the base cache tests and override for python-memcached until it's fixed.

Reply:
:+1: